[ADD] accounting/l10n_ca: canada accounting

samueljlieber opened the pull request:
Task: [#3809629](https://www.odoo.com/odoo/project/1625/tasks/3809629)

Adds Canada l10n accounting doc to the fiscal localizations.

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/9205).

samueljlieber commented:
https://github.com/odoo/documentation/commit/8ae1252e0053d1da0dd0071193009a9fca547f70 Added preliminary content [outlined in this task](https://www.odoo.com/odoo/project/1625/tasks/3809629) to the following sections:

- Modules installation
- Chart of accounts
- Fiscal positions
- Taxes
- AvaTax
- Reports
- Cash discount

samueljlieber commented:
https://github.com/odoo/documentation/commit/95d6b7debad937e5e4278d4f99d467bcdc69b671 Added a section for Payroll @larm-odoo

larm-odoo commented:
> [95d6b7d](https://github.com/odoo/documentation/commit/95d6b7debad937e5e4278d4f99d467bcdc69b671) Added a section for Payroll @larm-odoo

Hi Sam - per Catalina (cas): "I wouldn't add Payroll to it because we don't have any type of localized workflow for CA. If a Canadian customer would like to know more about Payroll the general doc should be enough"

So for now, I will not add that to the doc. If anything changes, I'll be sure to add the necessary info!

samueljlieber commented:
https://github.com/odoo/documentation/commit/b9adb683686ac880fd34bc164df622bad61c62dc
- Expanded on introduction
- added Canadian-specific report information
- expanded on writing checks
- removed payroll section
- expanded on receiving pre-authorized debits (stripe)

samueljlieber commented:
https://github.com/odoo/documentation/commit/1066a2d04db77c37cbc1755d310190d36e4b069d

- Listed default fiscal positions
- Added screenshot of fiscal positions

StraubCreative commented on content/applications/finance/fiscal_localizations/canada.rst:
```diff
@@ -0,0 +1,311 @@
+The Odoo Canada localization package provides tailored features and configurations for Canadian
+businesses.
+
+In addition, a series of videos on the subject of Accounting are available through Odoo's eLearning
+platform. These videos cover how to start from scratch, set up configurations, complete common
+workflows, and provide in-depth looks at some specific use cases, as well.
```
redundant + don't need 
```suggestion
A series of videos on the subject of Accounting are available through Odoo's eLearning platform.
These videos cover how to start from scratch, set up configurations, complete common workflows, and
provide in-depth looks at some specific use cases.
```

StraubCreative commented on content/applications/finance/fiscal_localizations/canada.rst:
```diff
@@ -0,0 +1,311 @@
+   * - :guilabel:`Canadian Checks Layout`
+     - `l10n_ca_check_printing`
+     - Enables the printing of payments on pre-printed check paper. Supports the three most common
+       check formats and will work out of the box with the linked checks from `checkdepot.net
```
future --> present tense, remove idiom
```suggestion
       check formats and works natively with the linked checks from `checkdepot.net
```

StraubCreative commented on content/applications/finance/fiscal_localizations/canada.rst:
```diff
@@ -0,0 +1,311 @@
+
+.. note::
+   When considering what taxes to be applied, it is the province where the delivery occurs that
+   matters. Therefore delivery is the responsibility of the vendor and is accounted for at the
```
need a comma there 😉 
```suggestion
   matters. Therefore, delivery is the responsibility of the vendor and is accounted for at the
```

StraubCreative commented on content/applications/finance/fiscal_localizations/canada.rst:
```diff
@@ -0,0 +1,311 @@
+------
+
+**Avalara AvaTax** is a cloud-based tax calculation and compliance software that integrates with
+Odoo for several localizations. Integrating AvaTax with Odoo provides real-time and region-specific
```
emphasizing what the reader cares about
```suggestion
Odoo for several localizations, including Canada. Integrating AvaTax with Odoo provides real-time and region-specific
```

StraubCreative commented on content/applications/finance/fiscal_localizations/canada.rst:
```diff
@@ -0,0 +1,311 @@
+tax calculations when items are sold, purchased, and invoiced in the database.
+
+.. important::
+   AvaTax is available for integration with databases/companies that have locations in Canada and
+   the United States. Reference the :ref:`avatax/fiscal_country` documentation for more information.
```
important detail to confirm
is the integration available for companies that **only** have locations in CA? Or is a US location required too?
```suggestion
   AvaTax is available for integration with databases/companies that have locations in Canada and/or
   the United States. Reference the :ref:`avatax/fiscal_country` documentation for more information.
```

StraubCreative reviewed (approved):
Hi @samueljlieber 
Looks good to me! I added a couple small comments for you to consider, however we can move to the next round with @odoo/localizations-doc-review for the sake of  deployment. Thank you for putting this together!

samueljlieber commented:
Thank your for your helpful review @StraubCreative! I implemented your suggestions in https://github.com/odoo/documentation/commit/3956fe42aa99ed225e1dd199fb04dd9dcb79093e.

samueljlieber commented:
@robodoo r+
